frontend/src/components/user-profile/profile.css

```diff
@@ -0,0 +1,96 @@
+body {
```

Comment:
in the future we will break it to smaller css because this is gonna serve us for other stuff as well. looks great